app/backend/src/couchers/servicers/requests.py

```diff
@@ -74,6 +75,15 @@ def CreateHostRequest(self, request, context):
             if request.to_date < least_current_date():
                 context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.DATE_TO_BEFORE_TODAY)
 
```

Comment:
The from date check is superfluous, since every time the from date is more than one year ahead, the to date is also more than one year ahead.

Reply:
What code are you referring to? Do you mean the next if statement? Why is that superfluous? Because of the if-statement that follows? That is checking for a to-date more that one year past the from-date.

Reply:
Sorry, line number got messed up on the phone :bow: I meant first check in the added lines.

But I also misread the code; you are right that the second tests is for the requested duration of the stay. Everything is fine, resolving conversation!